### PR TITLE
feat: Conversation template

### PR DESCRIPTION
**Pull request by XunchaoZ**

### CODE REVIEW

File: openllm-python/src/openllm/conversation.py [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+  print('\n')
+
+  print('-- Llama template --')
+  conv = get_conv_template('llama')
+  conv.set_system_message('You are a helpful, respectful and honest assistant.')
+  conv.append_message(conv.roles[0], 'Hello!')
+  conv.append_message(conv.roles[1], 'Hi!')
+  conv.append_message(conv.roles[0], 'How are you?')
+  conv.append_message(conv.roles[1], '')
+  print(conv.get_prompt())
```
Comment (aarnphm) [inline openllm-python/src/openllm/conversation.py:314]:
```suggestion
```

File: openllm-python/src/openllm/conversation.py [hunk 2]
```diff
@@ -0,0 +1,314 @@
+'''
```
Comment (aarnphm) [inline openllm-python/src/openllm/conversation.py:1]: can we make this one `_conversation.py` instead? Let's make it internal for now.

**Review by aarnphm — CHANGES_REQUESTED** [review]
Final comments. I assume you have tested with Llama @XunchaoZ?

File: openllm-python/src/openllm/conversation.py (hunk 2 above)
Comment (XunchaoZ) [reply openllm-python/src/openllm/conversation.py]: Done. For the llama testing I seem to encounter an error for the model-id checking. Can you check Slack?

**Comment by aarnphm** [comment]
@aarnphm cleanup

**Pull request merged**